src/xbps/troubleshooting/common-issues.md

```diff
@@ -2,14 +2,17 @@
 
 ## Verifying RSA keys
 
-If you are installing Void for the first time or the Void RSA key has changed,
-[xbps-install(1)](https://man.voidlinux.org/xbps-install.1) might report:
+If the Void RSA key has changed,
+[xbps-install(1)](https://man.voidlinux.org/xbps-install.1) will report the new
+key fingerprint and ask you to confirm it:
 
 ```
-<repository> repository has been RSA signed by <rsa_fingerprint>
+<repository> repository has been RSA signed by "Void Linux"
+Fingerprint: <rsa_fingerprint>
+Do you want to import this public key? [Y/n] y
```

Comment:
```suggestion
Do you want to import this public key? [Y/n]
```
Just a tiny suggestion, I would leave out the confirmation (y) in this case.

Reply:
Thanks, updated.